modules/images-using-customizing-s2i-images.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// * openshift_images/using_images/customizing-s2i-images.adoc
+
+[id="images-using-customizing-s2i-images-scripts-embedded_{context}"]
+= Invoking scripts embedded in an image
+
+Builder images provide their own version of the source-to-image (S2I) scripts that cover the most common use-cases. If these scripts do not fulfill your needs, S2I provides a way of overriding them by adding custom ones in the `.s2i/bin` directory. However, by doing this you are completely replacing the standard scripts. In some cases this is acceptable, but in other scenarios you can run a few commands before or after the scripts while retaining the logic of the script provided in the image. In this case, you can create a wrapper script that runs custom logic and delegates further work to the default script in the image.
+
+.Procedure
+
+. Look at the value of the `io.openshift.s2i.scripts-url` label to determine the location of the scripts inside of the builder image:
++
+[source,terminal]
+----
+$ podman inspect --format='{{ index .Config.Labels "io.openshift.s2i.scripts-url" }}' wildfly/wildfly-centos7
+image:///usr/libexec/s2i
+----
++
+You inspected the `wildfly/wildfly-centos7` builder image and found out that the scripts are in the `/usr/libexec/s2i` directory.
++
+. Invoke any of these scripts from your own by wrapping its invocation:
```

Comment:
Maybe "Create a script that includes an invocation of one of the standard scripts wrapped in other commands."